rest/src/main/java/org/jboss/pnc/rest/endpoint/BuildTaskEndpoint.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jboss.pnc.rest.endpoint;
+
+import com.wordnik.swagger.annotations.Api;
+import com.wordnik.swagger.annotations.ApiParam;
+import org.jboss.pnc.core.builder.coordinator.bpm.BpmCompleteListener;
+import org.jboss.pnc.spi.BuildStatus;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Api(value = "/build-tasks", description = "Build tasks.")
+@Path("/build-tasks")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public class BuildTaskEndpoint {
+
+    private static final Logger logger = LoggerFactory.getLogger(BuildTaskEndpoint.class);
+
+    @Context
+    private HttpServletRequest httpServletRequest;
+
+    private BpmCompleteListener bpmCompleteListener;
+
+    @Deprecated
+    public BuildTaskEndpoint() {} // CDI workaround
+
+    @GET
+    @Path("/{taskId}/completed")
+    public Response buildTaskCompleted(
+            @ApiParam(value = "Build task id", required = true) @PathParam("taskId") int taskId,
+            @ApiParam(value = "Build status", required = true) @QueryParam("buildStatus") BuildStatus buildStatus) {
+        bpmCompleteListener.notifyCompleted(taskId, buildStatus);
+        return Response.ok().build();
```

Comment:
BuildTask is not an entity and that is why BuildTaskEndpoint is not extending the AbstractEndpoint where fromEmpty is.